README.md

```diff
@@ -78,7 +78,10 @@ If you would like to work on this project full-time, [we are hiring!](https://ww
 
 ## Custom Rules
 
+To request new rules, please [contact us on our Community Forum](https://community.sonarsource.com/c/suggestions/).

```

Comment:
_last nitpick_ on line 41 we now have `Contact us on [our Community Forum]` (only `our community forum` is hyperlink), while here we have the whole text as link `[contact us on our Community Forum]`